src/codewhisperer/util/dependencyGraph/goDependencyGraph.ts

```diff
@@ -0,0 +1,236 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { Uri } from 'vscode'
+import { DependencyGraph, DependencyGraphConstants, Truncation } from './dependencyGraph'
+import * as CodeWhispererConstants from '../../models/constants'
+import { existsSync, readdirSync, statSync } from 'fs'
```

Comment:
try and use `const fs = FileSystemCommon.instance` instead since we will eventually need to run in the browser.

Reply:
Actually since only the async functions are available I'll leave it for now and revisit later